Docs/sphinx/Equations.rst

```diff
@@ -61,28 +59,26 @@ Some notes:
   all 3 components of the velocity. You should always initialize all velocity components to zero, and
   always construct the kinetic energy with all three velocity components.
 
-* There are ``NUM_ADV`` advected quantities, which range from :math:`{\tt
-  UFA: UFA+NUM\_ADV-1}`.  The advected quantities have no effect at all on
-  the rest of the solution but can be useful as tracer quantities.
+* There are ``NADV`` advected quantities, whose indices in the state vector range from :math:`{\tt UFA: UFA+nadv-1}`.  Here, ``UFA(=7)`` refers to the number of conserved variables not including the transported species. The advected quantities have no effect at all on the rest of the solution but can be useful as tracer quantities.
 
-* There are ``NUM_SPECIES`` species defined in the chemistry model, which range from :math:`{\tt UFS: UFS+NUM\_SPECIES-1}`.
+* There are ``NSPECIES`` species defined in the chemistry model, whose indices in the state vector range from :math:`{\tt UFS: UFS+nspecies-1}` where ``UFS=UFA+nadv``.
 
-* There are ``NUM_AUX`` auxiliary variables, from :math:`{\tt UFX:UFX+NUM\_AUX-1}`. The auxiliary variables are passed into the equation
-  of state routines along with the species.
+* There are ``NAUX`` auxiliary variables, from :math:`{\tt UFX:UFX+naux-1}`. Here, ``UFX=UFS+NUM_SPECIES``. The auxiliary variables are passed into the equation of state routines along with the species.
 
 * There are ``NUM_LIN`` linear passive variables, from :math:`{\tt ULIN:ULIN+NUM\_LIN-1}`. The linear passive variables are scalar variables where :math:`\mathbf{U}=\mathbf{Q}` instead of :math:`\mathbf{U}=\rho\mathbf{Q}`
 
 
 
+
 Primitive Forms
 ---------------
 
-PeleC uses the primitive form of the fluid equations, defined in terms of
+PeleC uses the primitive form of the inviscid fluid equations, defined in terms of
 the state :math:`\mathbf{Q} = (\rho, \mathbf{u}, p, \rho e, Y_k, A_k, B_k)`, to construct the
 interface states that are input to the Riemann problem. All of the primitive variables are derived from the conservative state
 vector. This task is performed in the routine ``ctoprim`` located in ``Src_nd/advection_util_nd.F90``.
```

Comment:
Can you correct this to remove the reference to Fortran? This function is now located in `Utilities.H`

There are a few more defunct references to Fortran that are still on this page in the documentation that we should remove as well.

Reply:
made the changes and reviewed it